compact/hcal/hcal_barrel_gdml.xml

```diff
@@ -0,0 +1,7682 @@
+<lccdd>
+
+    <limits>
+    </limits>
+
+    <regions>
+    </regions>
+
+    <display>
+    </display>
+
+    <materials>
+    <!-- EJ200 (Plastic scintillator) -->
+    <material name="EJ200" state="solid">
+      <D type="density" unit="g/cm3" value="1.023"/>
+      <fraction n="0.4757" ref="C"/>
+      <fraction n="0.5243" ref="H"/>
+    </material>
+    </materials>
+
+    <!-- Define detector -->
+    <detectors>
+      <documentation>
+      ### Barrel Hadronic Calorimeter
+      </documentation>
+      <detector
+	  id="oHCalBarrel_ID"
+	  name="oHcalBarrel"
+	  type="epic_oHcalBarrel"
+	  readout="oHcalBarrelHits"
+	  vis="HcalBarrelVis"
+	  env_vis="HcalBarrelEnvelopeVis"
+	  rmin="1767"
+	  rmax="2690"
+	  length="3170" 
```

Comment:
Not sure if you are using this to construct an envelope volume, but it should use the HCal parametrization volumes from definitions.xml. That allows us to apply any future shifts centrally without having to edit all detector files.

Reply:
Updated to use HCAL parametrization volumes

Reply:
Unfortunately this leads to overlaps because the rmin defined this way is too small for the HCAL sectors:

`    
<constant name="HcalBarrel_rmin"      value="ForwardServiceGap_rmax"/>
`

resolves to 195cm.  I believe that this is too small because in this region the plates in the HCAL are smaller radially, so the ForwardServiceGap_rmax is too large and near near the olenoid the plates protrude from the volume.  

I updated this to: 

`<constant name="HcalBarrel_rmin"      value="Solenoid_rmax"/>`

in definitions.xml.